README.md

```diff
@@ -70,4 +70,20 @@ navigation bar.
 A lot of components were refactored in custom Widgets, like CobbleCard, CobbleTile, CobbleButton, etc.
 and these components should serve you as building blocks upon which to build your UI. They are 
 showcased in WidgetLibrary screen and in golden (aka snapshot) tests. All golden images (how widgets 
-should look) are included in /test/components/goldens.
\ No newline at end of file
+should look) are included in /test/components/goldens.
+
+## Using localization
+
+To use localized string, add it to all `.json` files in `/lang`, start build_runner to generate 
```

Comment:
I would put a clarification statement to the "start build_runner" part. Maybe something like "See `Building mappings` section above"?